dms/tests/test_portal.py

```diff
@@ -6,6 +6,7 @@
 import odoo.tests
 
 
+@odoo.tests.tagged("post_install", "-at_install")
```

Comment:
Please don't do this. It's a big source for problems.

Reply:
Sorry, odoo suggested these change when is http test or warning show always.

Reply:
Weird... do you have more details about what warning it logs?

Reply:
Log message:
`odoo.addons.dms.tests.test_portal.TestDmsPortal: HttpCase test should be in post_install only`

https://github.com/odoo/odoo/blob/14.0/odoo/tests/common.py#L1377

Reply:
After internal discussion with @pedrobaeza, we're gonna obey that warning.

The problems introduced with post install mode were unavoidable in lower versions, but the framework has evolved and it seems like today it should be more reliable.

We'll revert this in the future if it gives those problems again.